docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
 
-    virtualenv venv-ivadomed
-    source venv-ivadomed/bin/activate
 
+Step 2: Install ``ivadomed``
+++++++++++++++++++++++++++++
 
-.. warning::
-   If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
 
-   ::
+    .. tabs::
 
-     virtualenv venv-ivadomed --python=python3.6
+        .. tab:: PyPI Installation
 
-Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            Install ``ivadomed`` and its requirements from
+            `PyPI <https://pypi.org/project/ivadomed/>`__:
 
+            ::
 
-::
+                pip install --upgrade pip
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+                pip install ivadomed
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        .. tab:: Repo Installation (Advanced or Developer)
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+            Bleeding-edge developments are available on the project's master branch
+            on Github. Install ``ivadomed`` from source:
 
-::
-   
-   pip install -r requirements_gpu.txt
+            ::
 
+                git clone https://github.com/ivadomed/ivadomed.git
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
+                cd ivadomed
 
+                pip install -e .
 
-::
 
-    pip install -r requirements.txt
+Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+    .. tabs::
 
-Step 4: Install from release (recommended)
-++++++++++++++++++++++++++++++++++++++++++
+        .. tab:: CPU Support
 
-Install ``ivadomed`` and its requirements from
-`Pypi <https://pypi.org/project/ivadomed/>`__:
+            If you plan to run ``ivadomed`` on CPU only, install pytorch per instructions `provided here for 1.8 <https://pytorch.org/get-started/previous-versions/#wheel>`_.
 
-::
+            Alternatively, if you have already downloaded/cloned the repo with access to the ``requirement.txt`` file, run the following command:
+            ::
 
-    pip install --upgrade pip
-    pip install ivadomed
+                pip install -r requirements.txt
 
-(Optional) Alternative Step 4 for Developers: Install from source
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Bleeding-edge developments are available on the project's master branch
-on Github. Installation procedure is the following at repository root:
+        .. tab:: Nvidia GPU Support
```

Comment:
> Alternatively, if you have already downloaded/cloned the repo with access to the ``requirement.txt`` file, run the following command:

Hmm... so, if the user is installing `ivadomed` using `pip install ivadomed` (which is the most prominent method in these new docs), then they won't have access to the requirements files, right? 

Because of that, I feel like the user might get stuck, especially for the GPU tab ("Where is `requirements_gpu.txt`? What do I install?"). And, it feels like a pretty big ask to expect the user clone the repo just to get access to `requirements_gpu.txt`. It kind of defeats the purpose of PyPI, aha. 

But, at the same time, I don't have a great solution to this, because the root cause is deeper than just documentation (https://github.com/ivadomed/ivadomed/issues/861).

Reply:
As a short-term fix for the docs, maybe we could explicitly list the `torch`/`torchvision` requirements from the requirements files:

CPU Only, Mac:

```rst
.. code::

   pip install torch==1.8.0 torchvision==0.9.0 --find-links https://download.pytorch.org/whl/torch_stable.html
```

CPU-only, Windows/Linux:

```rst
.. code::

   pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
```

GPU:

```rst
.. code::

   pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
```

This doesn't feel like a great solution, because we're duplicating information from the requirements files. But, addressing #861 probably deserves much more thought/discussion, and I understand that your plan was to finish this PR by today. :thinking:

Reply:
Good point. I mean, I feel like it almost feel better just to point user to the relevant section in the PyTorch installation guide than trying to duplicate information from them, even though technically the Torch website present these information more cleanly. I have adopted the above into a slightly more elaborate form including original solution.